modules/Bio/EnsEMBL/Compara/PipeConfig/CreateHmmProfiles_conf.pm

```diff
@@ -101,7 +101,7 @@ sub default_options {
         'clustering_max_gene_halfcount' => 750,
         # File with gene / peptide names that must be excluded from the
         # clusters (e.g. know to disturb the trees)
-        'gene_blacklist_file'           => '/dev/null',
+        'gene_blocklist_file'           => '/dev/null',
```

Comment:
What is the difference between blacklist and blocklist? blacklist should not be the right name?

Reply:
`blacklist` is a term that should be deprecated - it is a non-inclusive/harmful/outdated word that equates black with bad and white with good. It could be considered similar to the move from `master` to `main` in branch names.

Reply:
The community approved alternative is `blocklist`